docs/Common/Enums/index.md

```diff
@@ -336,6 +356,26 @@
 |PRESET_9|14||
 |CUSTOM_BUTTON|15||
 |SEARCH|16||
+Climate Buttons
```

Comment:
These category headers show up in the table as cells. They probably just need to be removed (Category can be stated in the description)